systemtests/tests/py3plug-fd-mariabackup/testrunner-default

```diff
@@ -35,6 +35,8 @@ start_test
 mysql_init
 mysql_server_start
 
+cp -v my.cnf ~/.my.cnf || :
+
```

Comment:
We should not do this.  People running our tests do not expect us to potentially destroy their configurations.  Lets see if this is really needed.